src/components/Tender.vue

```diff
@@ -0,0 +1,484 @@
+              <v-btn size=auto class="ml-2" icon color="transparent" variant="flat">
+                <v-tooltip activator="parent" location="top">
+                  Choose the deadline date for signed contract submission
+                </v-tooltip>
+                <v-icon icon="mdi-information-outline" class="inf-icon"></v-icon>
+              </v-btn>
+            </v-chip>
+            <v-text-field single-line color="blue" variant="outlined" v-model="tender.deadlineForSignedContract"
+              label="DeadLine for Signed Contract Submission" required density="compact" type="date">
+            </v-text-field>
+          </v-col>
+        </v-row>
+
+        <v-row>
+          <v-container class="d-flex align-center">
+            <p class="chapter-label">Documents</p>
+            <div class="horizontal-divider"></div>
+          </v-container>
+        </v-row>
+
+        <v-item-group class="py-5 mx-2">
+          <v-row class="mt-5 mx-8">
+            <v-item v-if="isContract">
+              <v-chip
+              size="large"
+              class="mb-6"
+              closable color="blue"
+              prepend-icon="mdi-file-document-multiple-outline"
+              label @click:close="isContract = false">
+              <div
+              id="text"
+              style="width: 50rem"
+              @click="openDialog(contract), dialog = true"
+              > {{ contract.name }} </div>
+              </v-chip
+              ></v-item>
+            <v-item v-if="!isContract">
+              <v-text-field
+              single-line label="* Contract"
+              variant="outlined"
+              density="compact"
+              ></v-text-field>
+              <input
+              input
+              id="contract-input"
+              class="d-none"
+              type="file"
+              accept="application/pdf"
+              @change="onFileChangedContract">
+              <v-btn
+              color="primary"
+              rounded="0"
+              height="40"
+              width="150">
+              <label
+              class="file-label"
+              for="contract-input"
+              >Upload</label>
+            </v-btn
+            ></v-item>
+          </v-row>
+
+          <v-row class="mt-5 mx-8">
+            <v-item v-if="isAwardDecision">
+              <v-chip
+              size="large"
+              class="mb-6"
+              closable
+              color="blue"
+              prepend-icon="mdi-file-document-multiple-outline"
+              label @click:close="isAwardDecision = false"
+              ><div
+              id="text"
+              style="width: 50rem"
+              @click="openDialog(awardDecision), dialog = true"
+              > {{ awardDecision.name }} </div>
+              </v-chip>
+            </v-item>
+            <v-item v-if="!isAwardDecision">
+              <v-text-field
+              single-line
+              label="* Award decision"
```

Comment:
1. Do not add * asterisk as a hardcoded plain text. Better to add isRequired field and bases on the field's value show or don't show it.
2. I'd implement  * asterisk as a CSS pseudo element. Smth like this https://stackoverflow.com/a/11197803